hummingbot/client/performance_analysis.py

```diff
@@ -201,5 +201,6 @@ def calculate_trade_performance(current_strategy_name: str,
     trade_performance_stats["portfolio_spent_quote_value"] = portfolio_spent_quote_value
     trade_performance_stats["portfolio_delta"] = portfolio_acquired_quote_value - portfolio_spent_quote_value
     trade_performance_stats["portfolio_delta_percentage"] = portfolio_delta_percentage
+    trade_performance_stats["no_of_trades"] = len(raw_queried_trades)
```

Comment:
In the #/2512 PR, I created a new module for performance calculation, it's `performance.py` and we'll not be using `performance_analysis` anymore. Though I believe you can get the same information from there.